tests/ut/test_decorators.py

```diff
@@ -395,26 +395,27 @@ def test_alias_takes_precedence(self, mock_cache):
 
     def test_get_cache_keys(self, decorator):
         keys = decorator.get_cache_keys(stub_dict, (), {"keys": ["a", "b"]})
-        assert keys == (["a", "b"], [], -1)
+        assert keys == (["a", "b"], ["a", "b"], [], -1)
 
     def test_get_cache_keys_empty_list(self, decorator):
-        assert decorator.get_cache_keys(stub_dict, (), {"keys": []}) == ([], [], -1)
+        assert decorator.get_cache_keys(stub_dict, (), {"keys": []}) == ([], [], [], -1)
 
     def test_get_cache_keys_missing_kwarg(self, decorator):
-        assert decorator.get_cache_keys(stub_dict, (), {}) == ([], [], -1)
+        assert decorator.get_cache_keys(stub_dict, (), {}) == ([], [], [], -1)
 
     def test_get_cache_keys_arg_key_from_attr(self, decorator):
         def fake(keys, a=1, b=2):
             """Dummy function."""
 
-        assert decorator.get_cache_keys(fake, (["a"]), {}) == (["a"], [["a"]], 0)
+        assert decorator.get_cache_keys(fake, (["a"]), {}) == ("a", ["a"], ["a"], 0)
```

Comment:
```suggestion
        assert decorator.get_cache_keys(fake, (["a"],), {}) == ("a", ["a"], ["a"], 0)
```

I think the expected output was correct before, someone just clearly messed up the input. The test passed before by luck, because `["a"][0] == "a"[0]`.